lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var cdiv = require( './../lib' );
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cdiv, 'function', 'main export is a function' );
+	t.end();
+});
+
+function testCdiv( a, b, c, d, e, f, label ) {
```

Comment:
Well, if you just need to do specific cases (or various value combinations), you could just use a single test; e.g.,

``` javascript
tape( 'the function performs complex division', function test( t ) {
    var expected;
    var v;

    v = cdiv( -13.0, -1.0, -2.0, 1.0 );
    expected = [ 5.0, 3.0 ];

    t.deepEqual( v, expected, 'returns expected values' );

    v = cdiv( 1.0e308, 5.0e307, 1.0, 0.5 );
    expected = [ 1.0e308, 0.0 ];

    t.deepEqual( v, expected, 'returns expected values' );

    // ...

    t.end();
});
```
As documentation, these are just various values you are using as test values. I don't need to be aware that you are trying to exercise various code paths.

More generally, we also like to compare our math implementations to reference implementations (e.g., Julia, Python, etc). In this case, we create test fixtures which contain (often) thousands of values (stored as JSON), which are loaded in the test file and then iterated against. For reference, see the various special functions in base which test after reference implementations.